README.md

```diff
@@ -1,8 +1,173 @@
-# Jaeger for iOS
-
 [![Build Status](https://travis-ci.org/wayfair/jaeger-swift.svg?branch=master)](https://travis-ci.org/wayfair/jaeger-swift)
```

Comment:
Why is this not working? 
<img width="217" alt="screen shot 2018-12-13 at 5 09 02 pm" src="https://user-images.githubusercontent.com/44030728/49970592-dba2ee00-fef9-11e8-89bf-2b80f7089766.png">

Reply:
there are no CI builds for the master branch. This will work once we merge develop into master.

Reply:
Thx!

Reply:
It was pointing to master. Updated this to point to `develop` for now. Will need to update this when we eventually merge to master.